crates/ui/src/dropdown.rs

```diff
@@ -161,7 +161,7 @@ where
 
         if let Some(item) = self.delegate.get(ix) {
             let list_item = ListItem::new(("list-item", ix))
-                .check_icon(IconName::Check)
```

Comment:
Here not need to change, we'd like to keep accepct IconName and Icon both.

Reply:
The IconName can into `Icon`.

Reply:
Thanks @huacnlee! I've changed it back to accept `IconName` as you suggested.

Reply:
Thanks